infrastructure/router/router.go

```diff
@@ -0,0 +1,31 @@
+package router
+
+import (
+	"WizelineApi/interface/controller"
+	//"academy-go-q42021/interface/controller"
+	"log"
+	"net/http"
+	"github.com/gorilla/mux"
+	"fmt"
+)
+
+
+
+func NewRouter() {
+	r := mux.NewRouter()
+
+	handler_pkm := controller.NewPokemonController()
+	//Pokemon Endpoints
+	r.HandleFunc("/pokemons", handler_pkm.GetAllPokemons).Methods("GET")
+	r.HandleFunc("/pokemon/{id}", handler_pkm.GetPokemon).Methods("GET")
```

Comment:
It's better to have the same route for common paths.
```suggestion
	r.HandleFunc("/pokemons/{id}", handler_pkm.GetPokemon).Methods("GET")
```